docker-compose.yml

```diff
@@ -31,10 +39,22 @@ services:
       - ./configuration/postgresql/initdb:/docker-entrypoint-initdb.d
       - postgresqldata:/var/lib/postgresql/data
 
+  postgresCsV4:
+    image: postgres:bookworm
```

Comment:
Is there a specific reason for using the bookworm image?

Reply:
I had issues with the migration script because of errors that i researched could be because of an older postgres version. After updating, i didn't get any errors. That is the main reason. Maybe it's not necessary, and I honestly didn't look futher into which image i should take instead - this seemed to work :-)